Added ⚡️ content to Designing Bitcoin Products > Principles (#494)
BitcoinDesign/Guide: guide/designing-products/principles.md

layout="float-left"
%}

### Enable import and export of wallet data, maximize backwards compatibility, and use open standards

</div>

Bitcoin is an open-source protocol, operating in a decentralized manner. This has led to a number of standards being developed to ensure compatibility between products. It should be easy to switch and move your Bitcoin wallet to a different application, should you wish. Ensuring that your product supports as many of these standards as possible is best practice and builds trust. More on [wallet interoperability]({{ '/guide/designing-products/wallet-interoperability/' | relative_url }}).

**Do**
- Support import and export of wallet data
    - For on-chain wallets, allow users to export and import wallets directly

>     - For Lightning wallets, make a clear path for the user to move their Lightning funds to another wallet should they choose

Reviewer: ```suggestion
    - For Lightning wallets, offer a clear path for the user to move their Lightning funds to another wallet
```
Don't think you need those last few words.

> - Support as many relevant BIPs and BOLTs as possible

Reviewer: Could link to BIPs in the Glossary page. Eventually, I assume, we'll also have something about BOLTs.

Author: OK, I just added a link to BIP in the glossary. I also opened a PR #496  to add BOLTs to the glossary. I did it separately in case there is a bunch of discussion about it. Perhaps we can debate that, get that PR merged, and then I can add the BOLT link here in this PR before merge.

Reviewer 2: its been merged @sbddesign

Author: Added link in 229b769
